### ISSUE TITLE
logout option

### ISSUE THREAD

**Issue by Skifpao** [tejado/pokemongo-api-demo#165]

**Comment by tejado** [comment]
A little bit more background?

**Comment by Skifpao** [comment]
way to logout from api. so it won't be like cached serverside if relogin in short time

**Comment by tejado** [comment]
Do you know how the logout is done in the app?

**Comment by Skifpao** [comment]
no, I don't know.

**Comment by genIterator** [comment]
There is no proto for that (currently?). 
The App probably just logs you in with the different account and lets the token of the other account run out.

**Comment by tejado** [comment]
Yes. There is also no login if you think it from the plain API side. You just execute something with the OAUTH token (google or ptc) and the request get redirected to some further server and there it get's re-executed.
I will change this thinking of the login process in pgoapi soon.

**Comment by xakiru** [comment]
Is there a way to retrive the token after we perform login, this way we can keep using it for all the future calls, until it expires, re-login for each client call is somehow suspicious.
Thanks
